Dockerfile.dev

```diff
@@ -0,0 +1,11 @@
+FROM node:6.10.1
+
+WORKDIR /usr/src/app
+
+ADD start.sh .
+
+RUN chmod +x ./start.sh
```

Comment:
Permissions are preserved and should be maintained in source. So change this outside the build.